rollup.config.js

```diff
@@ -55,6 +55,16 @@ export default [
       },
     ],
   },
+  {
+    input: 'src/index.ts',
+    plugins: plugins,
```

Comment:
Note: I _believe_ it’s generating the correct thing using the existing `tsconfig.json`. But best practice is to make a 2nd TS Config (e.g. `tsconfig.cjs.json`) if it stops working as-expected (worth noting that the current TSConfig uses legacy module resolution, so you may have to split it up if you used `"module": "ESNext", "moduleResolution": "Bundler":` which is now the recommended setup for most frontend code.